Including <linux/config.h> (via <sys/param.h> and <asm/param.h>) breaks the
build of busybox; it pollutes the namespace by undefining CONFIG_TR.
All architectures except Alpha appear fixed now.

--- include/asm-alpha/param.h.orig	2003-10-15 11:12:09.000000000 -0400
+++ include/asm-alpha/param.h	2003-11-01 17:46:06.000000000 -0500
@@ -5,7 +5,11 @@
    hardware ignores reprogramming.  We also need userland buy-in to the 
    change in HZ, since this is visible in the wait4 resources etc.  */
 
+#ifdef __KERNEL__
+/* Debian doesn't define CONFIG_ALPHA_RAWHIDE, and including
+   <linux/config.h> pollutes the namespace.  */
 #include <linux/config.h>
+#endif
 
 #ifndef HZ
 # ifndef CONFIG_ALPHA_RAWHIDE
